playbooks/adhoc/uninstall.yml

```diff
@@ -85,7 +85,7 @@
 
   - name: Remove packages
     package: name={{ item }} state=absent
-    when: not is_atomic | bool
+    when: not is_atomic | bool and openshift_remove_all | default(True) | bool
```

Comment:
This would mean we'd still need multiple blocks, is that still preferred? I'm open to it, no problem with me.